js/index.js

```diff
@@ -4,6 +4,15 @@ function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
 
   //... your code goes here
+  const price = product.querySelector('.price span').innerHTML;
+  const quantity = product.querySelector('.quantity input').value;
+  console.log(Number(price), Number(quantity));
```

Comment:
After you have confirmed the values, you should remove your `console.log` as it doesn't serve a purpose anymore